### DISCUSSION TITLE
What does a double lambda_method warning means?

### DISCUSSION THREAD

**Discussion by NvdGoorbergh** [dsharlet/LiveSPICE#157] [Q&A]
Hi, I just recently started with LTSPICe and then discovered livespice which I like a lot better.
The reason why I am getting this error is probably because I am doing something wrong but it's not entirely evident what.
I am trying to recreate the rev g2 from musikding.de and when I press simulate it either crashes or gives me an error after a while stateing that I didn't set the right parameters for the double lambda_method.

does this make sense to others?
With kind regards,
Nick

**Comment by Federerer** [comment]
do you have an input and output on your schematic?

**Reply by NvdGoorbergh** [reply]
I have an input and an speaker. Not entirely sure how I would assign an output besides the speaker.
I played with lowering the voltage (because when I set it to 9v it crashes.) I have a feeling its in the powersupply and maybe I am using the names wires wrong?
![afbeelding](https://user-images.githubusercontent.com/9550629/188284027-13daf99f-f589-4891-acc8-04ea40863f92.png)

**Comment by dsharlet** [comment]
This *might* have been fixed by #182, and the error should be a little bit better now (in the next release after Preview 14 or if you are building from master).
